### ISSUE TITLE
[1.11.2 - alpha-1.0.1] cascading worldgen lag crash

### ISSUE THREAD

**Issue by Boodaflow** [rwtema/DenseOres#143]
After starting up a new world, I started exploring near the spawn point until a sudden crash.
This issue seems to involve several mods including: Embers, Inventory Pets, AbyssalCraft, Recurrent Complex, Astral Sorcery, denseores, and Roguelike Dungeons.

[15:10:51] [Server thread/WARN]: denseores loaded a new chunk (0, 3  Dimension: 0) during chunk population, causing cascading worldgen lag. Please report this to the mod's issue tracker. This log can be disabled in the Forge config.

I tried posting the log in zip, but it wouldn't upload.

**Comment by Boodaflow** [comment]
capnkirok of Inventory-Pets seems to have the solution:

https://github.com/capnkirok/Inventory-Pets/issues/170

**Comment by Endmateria** [comment]
@Boodaflow No, somebody on this thread called me out saying that capnkirok's solution is wrong.  https://github.com/Vazkii/Quark/issues/412  These guys discuss the proper solution on this thread in detail.

**Comment by Boodaflow** [comment]
I'll relay the info to capnkirok.
